### PR TITLE
Item limit per cart added

### PR DESCRIPTION
**Pull request by Ansu-s**
I have removed this commit form the Media-API pull request and added it into this branch

### CODE REVIEW

File: src/main/java/com/example/bookstoreapp/models/CatalogItem.java [hunk 1]
```diff
@@ -14,16 +14,19 @@ public class CatalogItem {
   private String name;
   private Float price;
   private Integer stockQuantity;
+  private Integer itemLimit;
 
   public CatalogItemEntity toEntity() {
-    return new CatalogItemEntity(this.getId(), this.getName(), this.getPrice(), this.getStockQuantity());
+    Integer itemLimits = (this.getItemLimit() == null) ? 0 : this.getItemLimit();
```
Comment (rkpatra201) [inline src/main/java/com/example/bookstoreapp/models/CatalogItem.java:20]: itemLimits represent multiple itemLimit.
It should not be plural name.

**Review by rkpatra201 — COMMENTED** [review]
Reviewed

**Comment by rkpatra201** [comment]
@Ansu-s see the conflicts now. why we see conflict in item limit after merge of media api changes.

Media and item limit are 2 different tasks. Though both were in same branch earlier, it should not create such conflicts.

This clearly shows we have to know more about git. cc @Ansu-s / @Subhransupanda2000

File: src/main/java/com/example/bookstoreapp/models/CatalogItem.java (hunk 1 above)
Comment (Ansu-s) [reply src/main/java/com/example/bookstoreapp/models/CatalogItem.java]: fixed the variable name

**Pull request merged**